src/fidesops/service/connectors/query_config.py

```diff
@@ -256,6 +256,76 @@ def generate_update_stmt(
         returns None"""
 
 
+class ManualQueryConfig(QueryConfig[Executable]):
+    def generate_query(
+        self, input_data: Dict[str, List[Any]], policy: Optional[Policy]
+    ) -> Optional[ManualAction]:
+        """Describe the details needed to manually retrieve data from the
+        current collection.
+
+        Example:
+        {
+            "step": "access",
+            "collection": "manual_dataset:manual_collection",
+            "action_needed": [
+                {
+                    "locators": {'email': "customer-1@example.com"},
+                    "get": ["id", "box_id"]
+                    "update":  {}
+                }
+            ]
+        }
+
+        """
+
+        locators: Dict[str, Any] = self.node.typed_filtered_values(input_data)
+        get: List[str] = [
+            field_path.string_path
+            for field_path in self.node.node.collection.top_level_field_dict
+        ]
+
+        if get and locators:
+            return ManualAction(locators=locators, get=get, update=None)
```

Comment:
Is `update` sure to be `None` or does it get silently dropped?

Reply:
I'm not sure what you mean.  I was trying to make the ManualQueryConfig parallel our other QueryConfigs, like postgres and mongo. The `generate_query` method returns something that is GET-like, and the `generate_update_stmt` returns something that is UPDATE-like.  Because I can't run the query for them, this is something they need to complete manually, I have `generate_query` store GET instructions that I can surface to them, so here, the `update` is always None.  I'm just sharing the same resource here for caching them.

Reply:
I was asking if it is possible to have something passed like:

```py
{
    "step": "access",
    "collection": "manual_dataset:manual_collection",
    "action_needed": [
        {
            "locators": {"email": "customer-1@example.com"},
            "get": ["id", "box_id"],
            "update": {"authorized_user": None}
        }
    ]
}
```

In this case `"update": {"authorized_user": None}` would get changed to `None`. From your explanation it sounds like this couldn't happen so it's good.

Reply:
ok great thanks for clarifying